hspec-core/test/Test/Hspec/Core/RunnerSpec.hs

```diff
@@ -1021,3 +1021,15 @@ spec = do
     context "on failure" $ do
       it "returns False" $ do
         H.rerunAll config (Just report) result { specResultSuccess = False } `shouldBe` False
+
+  describe "timeout" $ do
```

Comment:
Please move this test case closer to tests for related options, e.g. around line 389.